packages/opc-timeline/src/opc-timeline.ts

```diff
@@ -0,0 +1,106 @@
+import { LitElement, html, property, customElement } from 'lit-element';
+import style  from './opc-timeline.scss';
+
+@customElement('opc-timeline')
+export class Timeline extends LitElement {
+  @property({ type: Array, attribute: 'steps'}) steps = [];
+  @property({ type: Number, attribute: 'current-step-index' }) currentStepIndex = 0;
+  @property({ type: String, attribute: 'variant'}) variant = 'default';
+  static get styles() {
+    return [ style ];
+  }
+
+  scrollHandler(direction) {
+    this.shadowRoot.querySelector('#timeline-steps').scrollBy({
+      left: direction === 'left' ?  -900 : 900,
+      behavior: 'smooth'
+    });
+  }
+
+  getDirectionArrow() {
```

Comment:
if this function is not an api then use _ as a prefix for function name. for eg: `_getDirectionArrow()`

Reply:
Makes sense did the same for the `scrollHandler` function as well